.github/workflows/main.yml

```
@@ -172,3 +172,45 @@ jobs:
       - name: Run tests
         run: |
           pytest -m 'vetiver'
+  
+  test-jupyter:
+    runs-on: ubuntu-latest
+    env: 
+      CONNECT_LICENSE: ${{ secrets.RSC_LICENSE }}
+      ADMIN_API_KEY: ${{ secrets.ADMIN_API_KEY }}
+    steps:
+      - uses: actions/checkout@v2
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - uses: extractions/setup-just@v1
+      - name: Build Containers
+        run: |
+            just integration-testing/build
+      - name: Start Connect + rsconnect-jupyter
+        run: |
+            just integration-testing/up
+        
+      - name: Run Cypress Tests
+        run: |
+            export ADMIN_API_KEY="${{ secrets.ADMIN_API_KEY }}"
+            just integration-testing/up-cypress
+
+      # Videos are captured whether the suite fails or passes
+      - name: Save videos
+        uses: actions/upload-artifact@v3
+        if: always()
```

This is the first time I've seen the use of `always()`, so I went to the documentation. There is a note about usage: https://docs.github.com/en/actions/learn-github-actions/expressions#always

Should this be changed so that it respects cancels?

```suggestion
        if: success() || failure()
```